administrator/components/com_admin/script.php

```diff
@@ -287,6 +287,8 @@ protected function uninstallExtensions()
              * ['type' => 'plugin', 'element' => 'demotasks', 'folder' => 'task', 'client_id' => 0, 'pre_function' => null],
              * ['type' => 'plugin', 'element' => 'compat', 'folder' => 'system', 'client_id' => 0, 'pre_function' => 'migrateCompatPlugin'],
              */
+            // Uninstall the old joomla 5 compatibility plugin
+            ['type' => 'plugin', 'element' => 'compat', 'folder' => 'system', 'client_id' => 0, 'pre_function' => null],
```

Comment:
```suggestion
            ['type' => 'plugin', 'element' => 'compat', 'folder' => 'behaviour', 'client_id' => 0, 'pre_function' => null],
```
Fix wrong folder in uninstallation of compat (5) plugin.